Description: Fix/update minor issues in upstream.
Author: Andreas B. Mundt <andi@debian.org>
Forwarded: https://sourceforge.net/p/atftp/bugs/8/
Last-Update: 2022-08-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/tftp_def.c
+++ b/tftp_def.c
@@ -54,7 +54,7 @@
      "Unknown transfer ID",
      "File already exists",
      "No such user",
-     "Failure to negotiate RFC1782 options",
+     "Failure to negotiate RFC2347 options",
 };
 
 
--- a/tftpd.c
+++ b/tftpd.c
@@ -1148,7 +1148,7 @@
                logger(LOG_INFO, "  bound to IP address %s only", tftpd_addr);
      }
      else
-          logger(LOG_INFO, "  started by inetd");
+          logger(LOG_INFO, "  started by inetd or socket activated");
      logger(LOG_INFO, "  logging level: %d", logging_level);
      if (trace)
           logger(LOG_INFO, "     trace enabled");
--- a/tftp.c
+++ b/tftp.c
@@ -868,34 +868,38 @@
      if (!data.connected)
           fprintf(stderr, "Not connected\n");
      else
-          fprintf(stderr, "Connected:  %s port %d\n", data.hostname,
+          fprintf(stderr, "Connected:   %s port %d\n", data.hostname,
                   data.port);
      fprintf(stderr, "Mode:       %s\n", data.tftp_options[OPT_MODE].value);
      if (data.verbose)
-          fprintf(stderr, "Verbose:    on\n");
+          fprintf(stderr, "Verbose:     on\n");
      else
-          fprintf(stderr, "Verbose:    off\n");
+          fprintf(stderr, "Verbose:     off\n");
      if (data.trace)
-          fprintf(stderr, "Trace:      on\n");
+          fprintf(stderr, "Trace:       on\n");
      else
-          fprintf(stderr, "Trace:      off\n");
+          fprintf(stderr, "Trace:       off\n");
      fprintf(stderr, "Options\n");
      if (data.tftp_options[OPT_TSIZE].specified)
-          fprintf(stderr, " tsize:     enabled\n");
+          fprintf(stderr, " tsize:      enabled\n");
      else
-          fprintf(stderr, " tsize:     disabled\n");
+          fprintf(stderr, " tsize:      disabled\n");
      if (data.tftp_options[OPT_BLKSIZE].specified)
-          fprintf(stderr, " blksize:   enabled\n");
+          fprintf(stderr, " blksize:    enabled\n");
      else
-          fprintf(stderr, " blksize:   disabled\n");
+          fprintf(stderr, " blksize:    disabled\n");
+     if (data.tftp_options[OPT_WINDOWSIZE].specified)
+          fprintf(stderr, " windowsize: enabled\n");
+     else
+          fprintf(stderr, " windowsize: disabled\n");
      if (data.tftp_options[OPT_TIMEOUT].specified)
-          fprintf(stderr, " timeout:   enabled\n");
+          fprintf(stderr, " timeout:    enabled\n");
      else
-          fprintf(stderr, " timeout:   disabled\n");
+          fprintf(stderr, " timeout:    disabled\n");
      if (data.tftp_options[OPT_MULTICAST].specified)
-          fprintf(stderr, " multicast: enabled\n");
+          fprintf(stderr, " multicast:  enabled\n");
      else
-          fprintf(stderr, " multicast: disabled\n");
+          fprintf(stderr, " multicast:  disabled\n");
 #ifdef HAVE_MTFTP
      fprintf(stderr, "mtftp variables\n");
      fprintf(stderr, " client-port:   %d\n", data.mtftp_client_port);
